### PR TITLE
Convert notebooks without images or wget to RST

### LINKED ISSUE #1667: notebook to rst where it's easy
Started looking at
- #1654

and was going to make a copy edit on proof-initiation, but realized it was a notebook. There are no graphics, nor any python code here, so we're not gaining anything with the notebook format. `nbconvert` should make this straightforward. See if there are any other candidates for easy conversion.

### PR DESCRIPTION
**Pull request by mccalluc**
- Fix #1667
- Replaces #1668
- Adds a helper script that handles the finicky translation of code to tabbed doctest.
- Minor manual code edits to work better as a doctest
- Take advantage of `dropdown` instead of relying on raw html `<details>`

Benefits:

-    Simpler source code
-    Stronger, more uniform tests. (Notebooks are run, but we don't check that the output is what we expect.)
-    Non deterministic behavior is more obvious, because we have `...`
-    Satisfy a prereq for adding R examples in tabs.

On this branch, there are no more matches:
```
grep 'image/png\|wget' -L source/**/*.ipynb
```
- images will be tricky.
- need to think more about the best way to optionally download data... wget in subprocess? wget from pypi? requests?

### CODE REVIEW

File: docs/source/api/user-guide/combinators/compositors.rst [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+-------
+
+Just like in chaining, since all compositors are
+“plain-old-measurements” they can also be used as arguments to
+interactive compositors. In this example, we nest a zCDP sequential
+compositor inside an approximate-DP sequential compositor.
+
+We first make the approximate-DP sequential compositor, accepting two
+queries. The first query must be :math:`(2 ε, 10^{-6} δ)`-DP, and the
+second :math:`(1 ε, 0 δ)`-DP.
```
Comment (Shoeboxam) [inline docs/source/api/user-guide/combinators/compositors.rst:256]: We should not use ε, δ as units. Commenting to open in new issue.

File: docs/source/api/user-guide/measurements/pca.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    .. tab-item:: Python
+        :sync: python
+
+        .. code:: python
+
+            >>> print(model.singular_values_)
+            [... ...]
+            >>> print(model.components_)
+            [[... ... ... ...]
+             [... ... ... ...]]
```
Comment (Shoeboxam) [inline docs/source/api/user-guide/measurements/pca.rst:148]: Nice job notating all of this.

File: docs/source/api/user-guide/measurements/randomized-response.rst [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+        :sync: python
+
+        .. code:: python
+
+            >>> def debias_randomized_response_bool(mean_release, p):
+            ...     """Adjust for the bias of the mean of a boolean RR dataset."""
+            ...     assert 0 <= mean_release <= 1
+            ...     assert 0 <= p <= 1
+            ...     
+            ...     return (mean_release + p - 1) / (2 * p - 1)
```
Comment (Shoeboxam) [inline docs/source/api/user-guide/measurements/randomized-response.rst:186]: We should move this into the library proper. Commenting for future issue.

File: docs/source/api/user-guide/measurements/randomized-response.rst [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+        .. code:: python
+
+            >>> def debias_randomized_response(mean_releases, p):
+            ...     """Adjust for the bias of the mean of a categorical RR dataset."""
+            ...     mean_releases = np.array(mean_releases)
+            ...     assert all(mean_releases >= 0) and abs(sum(mean_releases) - 1) < 1e-6
+            ...     assert 0 <= p <= 1
+            ...     
+            ...     k = len(mean_releases)
+            ...     return (mean_releases * (k - 1) + p - 1) / (p * k - 1)
```
Comment (Shoeboxam) [inline docs/source/api/user-guide/measurements/randomized-response.rst:270]: Similarly here, this should be a library utility. Future issue.

File: docs/source/contributing/rust-initiation.rst [hunk 5]
```diff
@@ -0,0 +1,382 @@
+OpenDP Rust Initiation
+======================
```
Comment (Shoeboxam) [inline docs/source/contributing/rust-initiation.rst:2]: This notebook needs to be updated. Future issue.

**Review by Shoeboxam — APPROVED** [review]
Thanks for converting these!

File: docs/source/api/user-guide/combinators/compositors.rst (hunk 1 above)
Comment (mccalluc) [reply docs/source/api/user-guide/combinators/compositors.rst:256]: - filed #1805

File: docs/source/api/user-guide/measurements/randomized-response.rst (hunk 3 above)
Comment (mccalluc) [reply docs/source/api/user-guide/measurements/randomized-response.rst:186]: - Filed #1806

File: docs/source/api/user-guide/measurements/randomized-response.rst (hunk 4 above)
Comment (mccalluc) [reply docs/source/api/user-guide/measurements/randomized-response.rst:270]: - Filed #1806

File: docs/source/contributing/rust-initiation.rst (hunk 5 above)
Comment (mccalluc) [reply docs/source/contributing/rust-initiation.rst:2]: - Filed #1807

**Comment by mccalluc** [comment]
Filed issues, resolved conflicts; Requesting re-review.

**Review by Shoeboxam — APPROVED** [review]
This looks great!

**Pull request merged**